TestCases/compliance-level-3/0063-feel-stddev-function/0063-feel-stddev-function-test-01.xml

```diff
@@ -0,0 +1,146 @@
+        <label>FEEL Functions: list</label>
+    </labels>
+
+    <testCase id="001">
+        <description>will return stddev</description>
+        <resultNode name="decision001" type="decision">
+            <expected>
+                <value xsi:type="xsd:decimal">2.0816659994661</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+<!--
+    <testCase id="002">
+        <description>will return median for odd list size</description>
+        <resultNode name="decision002" type="decision">
+            <expected>
+                <value xsi:type="xsd:decimal">6</value>
+            </expected>
+        </resultNode>
+    </testCase>
+-->
+
+    <testCase id="003">
+        <description>will error when no params</description>
+        <resultNode errorResult="true" name="decision003" type="decision">
+            <expected>
+                <value xsi:nil="true"/>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="003_a">
+        <description>will error when null param</description>
+        <resultNode errorResult="true" name="decision003_a" type="decision">
+            <expected>
+                <value xsi:nil="true"/>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="004">
+        <description>will error when list item not a number</description>
+        <resultNode errorResult="true" name="decision004" type="decision">
+            <expected>
+                <value xsi:nil="true"/>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="005">
+        <description>will error when list item not a number</description>
+        <resultNode errorResult="true" name="decision005" type="decision">
+            <expected>
+                <value xsi:nil="true"/>
+            </expected>
+        </resultNode>
+    </testCase>
+
+<!--
+    <testCase id="006">
+        <description>will median to decimals</description>
+        <resultNode name="decision006" type="decision">
+            <expected>
+                <value xsi:type="xsd:decimal">2.5</value>
+            </expected>
+        </resultNode>
+    </testCase>
+-->
+
+    <testCase id="007">
+        <description>will return null for empty list</description>
+        <resultNode errorResult="true" name="decision007" type="decision">
+            <expected>
+                <value xsi:nil="true"/>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="008">
+        <description>will error when a single item.  Note, the example given in the spec can only be calculated using
+            'sample' stddev algorithm, not the 'population' algorithm.  For the 'sample' algorithm, the stdev result
+            of a single item is undefined as part of the calculation is a division by the (sample size - 1).
+            Which, with a sample size of 1, results in a divide by zero error.
+        </description>
```

Comment:
Many thanks for pointing it out Greg!
I have raised an issue on the RTF to make this explicit, but I concur the example would lead to the sample standard deviation.

Reply:
:-) Thanks Matteo.